static/js/components/ranking_unit.tsx

```diff
@@ -93,26 +93,28 @@ function getRank(
 
 // Gets list of list of points to show where each list will be separated by ...
 // and each point will have the rank set.
-function getPointsList(props: RankingUnitPropType): RankingPoint[][] {
+export function getPointsList(
+  topPoints: RankingPoint[],
+  bottomPoints: RankingPoint[],
+  isHighest: boolean,
+  numDataPoints?: number
+): RankingPoint[][] {
```

Comment:
Nit: Can you add a comment doc above the function of what these parameters mean?

Reply:
done